larva/src/main/java/org/frankframework/larva/queues/QueueCreator.java

```diff
@@ -64,65 +70,58 @@ public Map<String, Queue> openQueues(String scenarioDirectory, Properties proper
 		List<String> jmsListeners = new ArrayList<>();
 		List<String> jdbcFixedQuerySenders = new ArrayList<>();
 
-		List<String> manuallyCreatedQueues = new ArrayList<>();
-
 		try {
 			// Use DirectoryClassLoader to make it possible to retrieve resources (such as styleSheetName) relative to the scenarioDirectory.
 			DirectoryClassLoader directoryClassLoader = new RelativePathDirectoryClassLoader();
 			directoryClassLoader.setDirectory(scenarioDirectory);
 			directoryClassLoader.setBasePath(".");
 			directoryClassLoader.configure(null, "LarvaTool");
 
-			Iterator iterator = properties.keySet().iterator();
-			while (iterator.hasNext()) {
-				String key = (String)iterator.next();
-				int i = key.indexOf('.');
-				if (i != -1) {
-					int j = key.indexOf('.', i + 1);
-					if (j != -1) {
-						String queueName = key.substring(0, j);
-						if(manuallyCreatedQueues.contains(queueName)) continue;
-						manuallyCreatedQueues.add(queueName);
-
-						debugMessage("queuename openqueue: " + queueName);
-						if ("org.frankframework.jms.JmsSender".equals(properties.get(queueName + ".className")) && !jmsSenders.contains(queueName)) {
-							debugMessage("Adding jmsSender queue: " + queueName);
-							jmsSenders.add(queueName);
-						} else if ("org.frankframework.jms.JmsListener".equals(properties.get(queueName + ".className")) && !jmsListeners.contains(queueName)) {
-							debugMessage("Adding jmsListener queue: " + queueName);
-							jmsListeners.add(queueName);
-						} else if ("org.frankframework.jdbc.FixedQuerySender".equals(properties.get(queueName + ".className")) && !jdbcFixedQuerySenders.contains(queueName)) {
-							debugMessage("Adding jdbcFixedQuerySender queue: " + queueName);
-							jdbcFixedQuerySenders.add(queueName);
-						} else {
-							String className = properties.getProperty(queueName+".className");
-							if(StringUtils.isEmpty(className)) continue;
-							Properties queueProperties = QueueUtils.getSubProperties(properties, queueName);
-
-							//Deprecation warning
-							if(queueProperties.containsValue("requestTimeOut") || queueProperties.containsValue("responseTimeOut")) {
-								errorMessage("properties "+queueName+".requestTimeOut/"+queueName+".responseTimeOut have been replaced with "+queueName+".timeout");
-							}
+			Set<String> queueNames = properties.keySet()
```

Comment:
Dit stuk is de werkelijke fix, de rest zijn wat code-cleanups. Scenarios zijn aangepast om te zien dat de aanpassingen ook daadwerkelijk werken, met namen met meer of minder punten.